packages/api/client.ts

```diff
@@ -144,32 +134,28 @@ export enum DataSource {
     tnds = "tnds",
 }
 
-export type ServicesQueryInput = {
+export type ServicesForOperatorQueryInput = {
     nocCode: string;
     dataSource: DataSource;
     modes?: string[];
 };
 
-export const getServices = async (dbClient: Kysely<Database>, input: ServicesQueryInput) => {
-    logger.info("Starting getServices...");
+export const getServicesForOperator = async (dbClient: Kysely<Database>, input: ServicesForOperatorQueryInput) => {
+    logger.info("Starting getServicesForOperator...");
 
-    let query = dbClient
+    return dbClient
         .selectFrom("services")
         .selectAll()
         .where("nocCode", "=", input.nocCode)
         .where("dataSource", "=", input.dataSource)
+        .$if(!!input.modes && input.modes?.length > 0, (qb) => qb.where("mode", "in", input.modes ?? []))
```

Comment:
do we need the chaining (modes?) if you are already checking it input.modes is present?